src/test/java/com/squareup/connect/api/TransactionsApiTest.java

```diff
@@ -140,7 +204,7 @@ public void listTransactionsTest() throws ApiException {
      * @throws ApiException
      *          if the Api call fails
      */
-    @Test
+    @Ignore
```

Comment:
Is there a reason why there are some tests ignored? If so, can you comment why instead of removing them?